static/app/views/issueDetails/groupRelatedIssues/index.tsx

```diff
@@ -1,12 +1,9 @@
-// XXX: A lot of the UI for this file will be changed once we use IssueListActions
```

Comment:
We decided to add the Open Issues button and let the developer take action in the Issue Stream directly since the action bar is very coupled in the main Issue Stream view.